README_python.md

```diff
@@ -62,7 +92,7 @@ with some differences where the C++ idiom is either inappropriate or impossible:
     Buffer<>(Type t, vector<int> extents, string name = "");
     ```
 
-    in Python, only the second variant is provided.
```

Comment:
Above, is written "variadic argumentlist" (missing space). GitHub's interface doesn't let me comment too far away from a change, ugh.

Reply:
Some of the entries below might need revision. For instance, "No mechanism is provided for overriding any runtime functions from Python." probably doesn't apply to the Python generators.

Reply:
Updated to clarify that this applies to JIT code, and added a section on the state of runtime functions for AOT code.